res/middleware/consoler-http.js

```diff
@@ -2,22 +2,30 @@
     var socket = io('http://' + document.location.host);
     var previousConsole = window.console || {};
     window.console = {
-        log:function(msg){
-            previousConsole.log && previousConsole.log(msg);
-            socket.emit('console','log', msg);
+        log:function(){
+            if(previousConsole.log) {
+                previousConsole.log.apply(this, arguments);
```

Comment:
This line fails when running from the browser, error: `Uncaught TypeError: Illegal invocation`